dev/breeze/doc/03_developer_tasks.rst

```diff
@@ -152,8 +152,7 @@ database client:
 You can change the used host port numbers by setting appropriate environment variables:
 
 * ``SSH_PORT``
-* ``WEBSERVER_HOST_PORT`` - for Airflow 2 web UI when --use-airflow-version is used
-* ``API_SERVER_HOST_PORT``
+* ``WEB_HOST_PORT`` - API server for Airflow 3, or webserver port for Airflow 2 when --use-airflow-version is used
```

Comment:
I chose this as a "least bad" option, so it's not specific to either Airflow version. I think `HTTP_HOST_PORT` would also be a good option. Happy to adjust if folks have strong opinions on this one.